prospector/datamodel/advisory.py

```diff
@@ -123,15 +123,28 @@ def extract_path_tokens(text: str) -> "list[str]":
     Returns:
         list: a list of paths that are found
     """
-    return [
-        re.split(r"\.|,|/", token.rstrip(r".,;:?!\"'"))
-        for token in text.split(" ")
-        if ("/" in token.rstrip(r".,;:?!\"'") and not token.startswith("</"))
-        or (
-            "." in token.rstrip(r".,;:?!\"'")
-            and token.rstrip(r".,;:?!\"'").split(".")[-1] in RELEVANT_EXTENSIONS
+    tokens = re.split(r"\s+", text)  # split the text into words
+    tokens = [
+        token.strip(",.:;-+!?)]}'\"") for token in tokens
+    ]  # removing common punctuation marks
+    paths = []
+    for token in tokens:
+        is_contains_path_separators = ("\\" in token) or ("/" in token)
+        is_period_separated = "." in token
+        has_relevant_extension = token.split(".")[-1] in RELEVANT_EXTENSIONS
+        is_xml_tag = token.startswith("<")
+        is_property = token.endswith("=")
+
+        # if strict_extensions:
```

Comment:
move this explanation up, in the fucntion doc block

Reply:
Good point. I missed that. I will do this together with the other fixes.

Reply:
I guess this is the only necessary change for this PR, then we can merge and move on.